context/app/static/js/components/detailPage/entityHeader/WorkspaceMenu/WorkspaceMenu.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useRef } from 'react';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+
+import { SvgIcon } from '@material-ui/core';
+import { ReactComponent as WorkspacesIcon } from 'assets/svg/workspaces.svg';
+
+import { WhiteBackgroundIconButton } from 'js/shared-styles/buttons';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+
+const componentId = 'workspaces-menu';
+
+function WorkspaceMenu({ entity_type, uuid }) {
+  const anchorEl = useRef(null);
+  const [menuIsOpen, setMenuIsOpen] = useState(false);
+
+  return (
+    <>
+      <SecondaryBackgroundTooltip title="Launch a new workspace in Jupyter notebook.">
```

Comment:
We don't know how long it's going to be until it does this. Suggest something like
```suggestion
      <SecondaryBackgroundTooltip title="Download Jupyter notebook.">
```

Reply:
and when we do get there, I feel like the notebook is in the workspace, and not the other way around.

Reply:
"Download Jupyter notebook." should be better for this context, the language might change later when the workspace is implemented.